src/AccessibilityInsights.SharedUx/Controls/EventRecordControl.xaml.cs

```diff
@@ -364,6 +400,26 @@ public void SaveEvents()
             MessageDialog.Show("There is no event to save.");
         }
 
+        /// <summary>
+        /// Check whether there is any event recorded
+        /// </summary>
+        /// <returns></returns>
+        private bool HasRecordedEvents()
+        {
+            return this.dgEvents.Items != null && this.dgEvents.Items.Count != 0;
+        }
+
+        /// <summary>
+        /// Save Event Messages in Json format
+        /// </summary>
+        /// <param name="path"></param>
+        private void SaveInJson(string path)
```

Comment:
>SaveInJson [](start = 21, length = 10)

This could use SetupLibrary.FIleHelpers.SerializeDataToJSON (same method that ultimately handles most other config serialization)